ios/FluentUI/Table View/TableViewHeaderFooterView.swift

```diff
@@ -39,20 +39,14 @@ open class TableViewHeaderFooterView: UITableViewHeaderFooterView, TokenizedCont
                                    containerWidth: CGFloat = .greatestFiniteMagnitude,
                                    accessoryView: UIView? = nil) -> CGFloat {
         let tokenSet: TableViewHeaderFooterViewTokenSet = .init(style: { style }, accessoryButtonStyle: { AccessoryButtonStyle.regular })
-        let verticalMargin: CGFloat
         let font = tokenSet[.textFont].uiFont
-        switch style {
-        case .header, .footer:
-            verticalMargin = TableViewHeaderFooterViewTokenSet.titleDefaultTopMargin + TableViewHeaderFooterViewTokenSet.titleDefaultBottomMargin
-        case .headerPrimary:
-            verticalMargin = TableViewHeaderFooterViewTokenSet.titleDefaultTopMargin + TableViewHeaderFooterViewTokenSet.titleDefaultBottomMargin
-        }
+        let verticalMargin = TableViewHeaderFooterViewTokenSet.titleDefaultTopMargin + TableViewHeaderFooterViewTokenSet.titleDefaultBottomMargin
 
         if let accessoryView = accessoryView {
             accessoryView.frame.size = accessoryView.systemLayoutSizeFitting(CGSize(width: containerWidth, height: .infinity))
         }
 
-        let titleWidth = containerWidth - (TableViewHeaderFooterViewTokenSet.horizontalMargin + TableViewHeaderFooterView.titleTrailingOffset(accessoryView: accessoryView) + TableViewHeaderFooterView.titleLeadingOffset())
+        let titleWidth = containerWidth - (TableViewHeaderFooterView.titleLeadingOffset() + TableViewHeaderFooterView.titleTrailingOffset(accessoryView: accessoryView))
```

Comment:
`titleLeadingOffset` being a private static func seems odd to me, but not really an issue

Reply:
I think that's because the height func is a class func and cannot take instance funcs.